apps/cli/src/compose/rollups/proxy/rollups-node.yaml

```diff
@@ -0,0 +1,17 @@
+http:
+    routers:
+        inspect_server:
+            rule: "PathPrefix(`/inspect`)"
+            service: inspect_server
+        rpc_server:
+            rule: "PathPrefix(`/rpc`)"
+            service: rpc_server
+    services:
+        inspect_server:
+            loadBalancer:
+                servers:
+                    - url: "http://rollups-node:10012/inspect"
+        rpc_server:
+            loadBalancer:
+                servers:
+                    - url: "http://rollups-node:10011/inspect"
```

Comment:
```suggestion
                    - url: "http://rollups-node:10011/rpc"
```

```suggestion
                    - url: "http://rollups-node:10011/json-rpc"
```

Reply:
My experience from other projects is that `rpc` is more common than `json-rpc` for endpoints

Reply:
Do you think we should use a `/rollups/inspect` adn `/rollups/rpc` to hide/abastract the port?

Or even `/cartesi/node/rpc`, `/cartesi/node/inspect`

Reply:
I'd leave as it is for simplicity. Node is the main service of the environment, it's why the environment exist, it deserves the "root path"